integration/simulation/output_stats.go

```diff
@@ -41,12 +41,12 @@ func (o *OutputStats) countRollups() {
 	l2Node := o.simulation.l2Network.nodes[0]
 
 	// iterate the Node Headers and get the rollups
-	for header := l2Node.DB().GetCurrentRollupHead(); header != nil; header = l2Node.DB().GetRollupHeader(header.Parent) {
+	for header := l2Node.DB().GetCurrentRollupHead(); header != nil && header.ID != obscurocommon.GenesisHash; header = l2Node.DB().GetRollupHeader(header.Parent) {
```

Comment:
We should probably only have one exit condition :)

Reply:
for some reason this is the condition that works correctly.